shared/chat/inbox/index.desktop.tsx

```diff
@@ -124,16 +124,22 @@ class Inbox extends React.Component<T.Props, State> {
       return
     }
     if (!this.props.unreadIndices.length || this.lastVisibleIdx < 0) {
-      this.setState(s => (s.showUnread ? {showUnread: false} : null))
+      if (this.state.showUnread) {
```

Comment:
only setting if the state is old. calling setState and having it bail with null still causes the 'cascading effects' warnings in devtools. i think this is fine and likely won't ever race, you'll just possibly queue up ones that'll be replaced later which is fine